.env.example

```
@@ -3,23 +3,25 @@
-LOGS2LOGFIRE=false                                       # Post all logs to Logfire. If false, some logs are written to a local log file.
+MAX_RESEARCH_LOOPS=10                                    # Number of search-summary-reflection loops
+MAX_WEB_SEARCH_RESULTS=5                                 # Number of results in a single web search
 TEMPERATURE_QUERY=1.0                                    # Temperature for the model generating the web queries
 TEMPERATURE_SUMMARY=1.0                                  # Temperature for the model generating the summaries of the web search results
 TEMPERATURE_REFLECTION=1.0                               # Temperature for the model generating the reflection on the summaries
 TEMPERATURE_FINAL_SUMMARY=1.0                            # Temperature for the model generating the final summary of the research report
+LOGS2LOGFIRE=false                                       # Post all logs to Logfire. If false, some logs are written to a local log file.
+REPORTS_FOLDER="reports/"                                # Output directory for the final reports
+
+# URLs
+SEARXNG_HOST="http://localhost:8080"                     # SearXNG host URL, e.g. http://localhost:8080 or http://host.docker.internal:8080 (in dev container)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)